bot/exts/moderation/modpings.py

```diff
@@ -27,11 +27,11 @@ def __init__(self, bot: Bot):
     async def normalize_roles(self) -> None:
         """Reschedule moderators role re-apply times."""
         await self.bot.wait_until_guild_available()
-        await self._role_scheduler.wait_until_ready()
-
         self.guild = self.bot.get_guild(Guild.id)
         self.moderators_role = self.guild.get_role(Roles.moderators)
 
+        await self._role_scheduler.wait_until_ready()
+
```

Comment:
I don't see how moving it down after getting the role and guild makes a difference.